students/BrianE/session02/GridPrinter.py

```diff
@@ -0,0 +1,43 @@
+"""
+GridPrinter.py
+
+Prints ascii grid based on desired box size and grid size.
+"""
+
+
```

Comment:
Don't forget to implement the initial grid which only takes 1 parameter.

Reply:
I've added an example of how my function supports a single parameter (box_size). The default value of 2 for grid_size supports calling grid(5). 

Let me know if you want a completely separate function with only a single parameter defined and I'll add that, instead.

Reply:
Went ahead and added the initial print_grid() function just in case.